js/usernameSelection.js

```diff
@@ -43,19 +42,21 @@ const addUsername = function addUsername() {
 };
 
 const addOneUsername = function addOneUsername(username) {
-    USERNAMES.push(username);
+    let usernames = JSON.parse(localStorage.getItem('usernames'));
+    usernames.push(username);
+    localStorage.setItem('usernames', JSON.stringify(usernames));
     content.innerHTML = '';
     getData();
     renderUsernames();
 };
 
-const inputOptions = new Promise((resolve) => {
-    setTimeout(() => {
-        resolve(USERNAMES);
-    }, 2000);
-});
-
 const removeUsername = function removeUsername() {
+    inputOptions = new Promise((resolve) => {
+        let usernames = JSON.parse(localStorage.getItem('usernames'));
+        setTimeout(() => {
```

Comment:
I got that code snippet from here: https://limonte.github.io/sweetalert2/
I guess it times out if you don't put in any value.